base_encrypted_field/models/res_users.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    pgp_private_key = fields.Text('Private pgp key', readonly=True)
```

Comment:
Both keys are stored on the server, but there should be a password on the private key which only the client knows. All of the crypto methods are taking place client side, so the password never touches the server.